### ISSUE TITLE
Mac need a few tweaks to CMake to work out of the box

### ISSUE THREAD

**Issue by ajc158** [BRAHMS-SystemML/brahms#12]
brahms-execute should be moved to the same dir as brahms, and brahms should be made executable

**Comment by sebjameswml** [comment]
They should already go in the same directory; in framework/execute/CMakeLists.txt I have:
```
# This means install the brahms-execute target into BIN_INSTALL_PATH (making it executable)
install(TARGETS brahms-execute DESTINATION ${BIN_INSTALL_PATH})
# This is similar, but uses PROGRAMS as it's a script - should also be executable and end up in the same location.
install(PROGRAMS brahms DESTINATION ${BIN_INSTALL_PATH})
```
So I'm puzzled that you find they don't. I'll fire up the Mac at home at some point...

**Comment by ajc158** [comment]
Hmm... they have the following structure
`.../bin/
  brahms-execute
  /brahms/
    brahms
    <other brahms libs>`

**Comment by sebjameswml** [comment]
Hi Alex, is this still an issue?

**Comment by sebjameswml** [comment]
Yes, it is - I've just observed it on Alex Blenkinsop's installation.

**Comment by ajc158** [comment]
I thought I created a pull request as you own the rep

**Comment by sebjameswml** [comment]
I merged your pull request which was quite old. This issue isn't yet fixed, but I know what the problem is.

**Comment by sebjameswml** [comment]
The problem with the brahms script appearing in a subdirecory on a STANDALONE build should be fixed with d063542

**Comment by sebjameswml** [comment]
I reckon this issue is sorted.
